Sources/Plasma/Apps/plClient/external/makeres.py

```diff
@@ -72,16 +73,18 @@
 
 	## Do the work!
 	if options.render:
-		ret = subprocess.call(["python", os.path.join(inpath, "render_svg.py"), "-i", inpath, "-o", os.path.join(workpath, "render")], stdout=sys.stdout, stderr=sys.stderr)
+		ret = subprocess.call([sys.executable, os.path.join(inpath, "render_svg.py"), "-i", inpath, "-o", os.path.join(workpath, "render")], stdout=sys.stdout, stderr=sys.stderr)
```

Comment:
We should probably use `sys.executable` for the subprocess call to `create_resource_dat.py` as well.